R/pprimarycensored.R

```diff
@@ -99,24 +99,17 @@
 #' )
 pprimarycensored <- function(
     q, pdist, pwindow = 1, D = Inf, dprimary = stats::dunif,
-    dprimary_args = list(), pdist_name = NULL, dprimary_name = NULL, ...) {
+    dprimary_args = list(),
+    pdist_name = deprecated(), dprimary_name = deprecated(), ...) {
```

Comment:
I think this is a hard depreciation but we need a soft one i.e. where the functionality continues to work for the user but with a warning for at least a version

Reply:
> `r lifecycle::badge("deprecated")` this argument will be
#' ignored; use [attach_distribution_name()] on `dprimary` instead

This is what makes me think its a hard dep

Reply:
i'll switch to slower deprecation protocol.